deploy-board/deploy_board/webapp/env_views.py

```diff
@@ -1952,3 +1954,19 @@ def override_session(request, name, stage):
     session_num = request.GET.get('session_num')
     schedules_helper.override_session(request, name, stage, session_num)
     return HttpResponse(json.dumps(''))
+
+def get_cluster_name(request, name, stage):
+    cluster_name = '{}-{}'.format(name, stage)
+    current_cluster = clusters_helper.get_cluster(request, cluster_name)
+    if current_cluster is None:
+        env = environs_helper.get_env_by_stage(request, name, stage)
```

Comment:
nit: it looks like for a couple of places where the new methods are called, the env is already available before the method is called. Could we pass in the env instead, and save an api call? Something like
```suggestion
def get_cluster_name(request, name, stage, env=None):
    cluster_name = '{}-{}'.format(name, stage)
    current_cluster = clusters_helper.get_cluster(request, cluster_name)
    if current_cluster is None:
        if env is None:
            env = environs_helper.get_env_by_stage(request, name, stage)
```

Reply:
yeah good call

Reply:
updated